app/Main.hs

```diff
@@ -297,108 +101,92 @@ instance HasEntryPoint CallsOptions where
 instance HasEntryPoint CallGraphOptions where
   getEntryPoint root = EntryPoint root . pure . (^. graphInputFile)
```

Comment:
Can we move all these instances somewhere else, maybe to `Command.hs`?

Reply:
That would be ok to do.

Also, I think we should consider moving the input files to be global arguments instead of command specific

Reply:
Good Idea!